packages/frontend/web-ui/src/home/hooks/useGameStatus.jsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { httpClient } from '../../common/http/services/HttpService';
+
+const STATUS_GAME_IDLE = 'idle';
+const STATUS_GAME_PENDING = 'pending';
+export const STATUS_GAME_FULFILLED = 'fulfilled';
+export const STATUS_GAME_REJECTED = 'rejected';
+
+const UNEXPECTED_ERROR_MESSAGE = 'Unexpected error!';
+
+export const useGameStatus = () => {
```

Comment:
As said, this should receive an status parameter to it can be invoked twice to fetch non started and active games

Reply:
I'll check it!